src/utils/junitUtils.ts

```diff
@@ -0,0 +1,76 @@
+import { SendJsonOutput } from '@/cli/send/jsonOutput';
+
+const xmlSpecialCharReplacementMap: Record<string, string> = {
```

Comment:
I already have [xmldom reference](https://github.com/AnWeber/httpyac/blob/main/package.json#L99) in the project. I am considering building this string concatenation story, with the framework and then also [outputting it using with `formatXML`](https://github.com/AnWeber/httpyac/blob/main/src/plugins/xml/xmlResponseInterceptor.ts#L3).
I would compare it once to see if it reads better, but it would probably solve the escaping issues directly.

Reply:
To me xmldom looks nicer. Then you don't have to worry about XML output at the same time.